src/hidimstat/_utils/utils.py

```diff
@@ -105,3 +138,40 @@ def seed_estimator(estimator, random_state=None):
                 setattr(value, "random_state", RandomState(rng.bit_generator))
 
     return estimator
+
+
+def check_statistical_test(test):
```

Comment:
Is it possible to merge #460, which looks almost ready, before this and set NB-ttest as the default? 
That would avoid merging in a default argument that we want to replace later. 

Then, I would suggest moving `check_statistical_test` to `/statistical_tools/statistical_test`